app/lib/common/store/Colors.dart

```diff
@@ -4,4 +4,5 @@ class AppColors {
   static const Color primaryColor = Color(0xffE9385D);
   static const Color textFieldColor = Color(0xff171717);
   static const Color backgroundColor = Color(0xff0A0A0A);
+  static const Color white = Colors.white;
 }
```

Comment:
This needs to get updated as we don't want to define seperate classes for colors and styles. Rather have single class to handle theme overall. Already mentioned this in #79. For now its okay but open a seperate PR for this.